### DISCUSSION TITLE
меню

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#517] [General]
...

**Comment by Kor-Angar** [comment]
а в MDK тут две текстурки используются

```D
//
// Statische Items
//

INSTANCE MENUITEM_MAIN_HEADLINE2(C_MENU_ITEM_DEF) 
{
	backPic		= "menu_gothic.tga";
	// Positionen  und Dimensionen	
	posx		=  250;		posy		=  500;	
	dimx		= 7900;		dimy		= 2300;	
	// Weitere Eigenschaften
	alphaMode 		= "BLEND";	
	alpha			= 255;		
	flags			= flags | IT_TXT_CENTER;
	flags			= flags & ~IT_SELECTABLE;
};

INSTANCE MENUITEM_MAIN_HEADLINE(C_MENU_ITEM_DEF) 
{
	backPic		= "menu_gothicshadow.tga";
	// Positionen  und Dimensionen	
	posx		=  250;		posy		=  500;
	dimx		= 7900;		dimy		= 2300;				
	// Weitere Eigenschaften
	alphaMode		= "BLEND";	
	alpha			= 220;			
	flags			= flags | IT_TXT_CENTER;
	flags			= flags & ~IT_SELECTABLE;
};
```

**Reply by dosinabox** [reply]
я делаю основу и тень на первой, чтобы вторую оставить пустой и сэкономить 256 КБ

**Reply by Kor-Angar** [reply]
так у тебя другая тектсурка?

**Reply by dosinabox** [reply]
у меня эта, с парой правок и совмещенная вместе с тенью

**Reply by Kor-Angar** [reply]
скинь плиз

**Reply by dosinabox** [reply]
> скинь плиз

мою текстуру из обновления?

**Reply by Kor-Angar** [reply]
да

**Reply by dosinabox** [reply]
В оригинальном релизе Г2НВ на дисках от Акеллы этой текстуры не было, забыли добавить. Её потом выкладывали на форуме отдельно. И там были тени от "блеска" на буквах Н и А. Я эти лишние тени исправила и потом добавляла эту текстуру в неофициальное обновление и стим-фикс. Постепенно она добралась до официального контента русской стим-версии, наконец оказавшись на своем месте.
[menu_gothic_ru.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/11391349/menu_gothic_ru.zip)

**Comment by Kor-Angar** [comment]
а кто нибудь пробовал делать интро для меня?

`//	backWorld	=	"cutscenes\INTRO.zen";`

**Reply by dosinabox** [reply]
не знаю, я нет

**Comment by Kor-Angar** [comment]
в меню героя есть "круги магии" и есть "создание рун".

если герой учит круги магии, то он умеет создавать руны. то есть герой не может учить круги не умея создавать руны. поэтому создание рун в принципе можно и убрать ;-)
